# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../hermes/destdir.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- Hermes-1.3.3/src/Makefile.am.vanilla	2007-06-24 19:20:39.521221496 +0200
+++ Hermes-1.3.3/src/Makefile.am	2007-06-24 19:20:59.150340096 +0200
@@ -32,9 +32,9 @@
 	cp $@ `echo $@ | sed -e 's/\.lo$$/.o/'`
 
 install-data-local:
-	$(INSTALL) -m 0755 -d $(includedir)/Hermes
+	$(INSTALL) -m 0755 -d $(DESTDIR)$(includedir)/Hermes
 	for incfile in $(publicheaders); do \
-	  $(INSTALL) -m 0644 $$incfile $(includedir)/Hermes; \
+	  $(INSTALL) -m 0644 $$incfile $(DESTDIR)$(includedir)/Hermes; \
 	done
 
 uninstall-local:
